k8s/tools/check-lockfile.py

```diff
@@ -15,11 +15,14 @@ def main():
     timeout = values.get('timeout')
     if timeout > max_timeout:
       max_timeout = timeout
+  config.log.info(f'Set max timeout: {max_timeout}')
```

Comment:
Does config.log actually work?  Why not import logging and use getlogger?

Reply:
I don't see why it wouldn't since the config module uses getlogger ```log = logging.getLogger('turbinia')
```. Do we need to repeat that statement here?

Reply:
Made some changes to this.

Reply:
cool, yeah, I think it's a little more idiomatic like this.